docs/python/programming/math/functions.rst

```diff
@@ -28,7 +28,7 @@ Diagnostics:
 
         .. note::
 
-            In Python code, use Python's ``abs`` function, which works on both numbers and numpy arrays:
+            In Python code, use Python's ``abs`` function, which works on both numbers and numpy arrays, as well as Vectors (Vectors do not print their contents) :
```

Comment:
why are we adding this caveat? Also the English has gotten weird, abs works on numbers, numpy arrays, and Vectors. Saying both A and B and also C is not the best.

As far as the example below, I'd say either:

```python
>>> v = h.Vector([1, 6, -2, -65])
>>> abs(v).printf()
1	6	2	65	
4
```
or

```python
>>> v = h.Vector([1, 6, -2, -65])
>>> list(abs(v))
[1.0, 6.0, 2.0, 65.0]
```